src/jloptions.c

```diff
@@ -505,6 +507,20 @@ JL_DLLEXPORT void jl_parse_opts(int *argcp, char ***argvp)
             else
                 jl_errorf("julia: invalid argument to --handle-signals (%s)", optarg);
             break;
+        case 'r':
+            {
+                int rpc = strtol(optarg, &endptr, 10);
+                if (errno != 0 || optarg == endptr) {
+                    jl_errorf("julia: -r,--region-pg-cnt=<n> must be an integer >= 1");
+                } else if (*endptr == 'm') {
+                    jl_options.region_pg_cnt = 64 * rpc;
```

Comment:
It's a little weird that the optional is called page count but the value is in size.